README.md

```diff
@@ -2,208 +2,269 @@
+make functional
+```
 
 ## Deployment
 
-This application is deployed via Github Actions to an ECS cluster. Once merged into QA changes are deployed to [the DRB QA Instance](http://drb-api-qa.nypl.org)
+This application deploys via Github Actions to AWS ECS:
 
-Production deployments are to be made when releases are cut against `main`.
+- QA changes deploy automatically to [DRB QA Instance](http://drb-api-qa.nypl.org)
+- Production deploys via releases cut against `main`
 
 ### Release Process
 
-We use git tags to tag releases and github's release feature to deploy.  The steps are as follows:
-
-  1. Decide on a new version number (assume 0.12.0 for the following steps)
-  2. Make sure your local `main` branch is up to date
-  3. Create a new branch named as the version number
-  4. Commit your change and push to your branch. 
-  5. Create a PR titled the Release <version number>
-  6. Once the PR is approved, squash and merge the change into main.
-  7. In github, navigate to the 'Releases' tab and click on 'Draft a new release'
-  8. Choose to create a new tag (e.g. v0.12.0), set `main` as the target, and name your release after the new version number
-     ('v0.12.0')
-  9.  Use the following command to print out the release notes: `cat <(git log <first-commit>..HEAD --pretty=format:"%h - %s (%an, %ad)" --date=short)`. Replace `<first-commit>` with the SHA of the first commit.
-  10. Give the team a heads up in  `#researchnow_aka_sfr` in Slack, make sure the release is 'Set as the latest release', and hit 'Publish release'
-  11. Check the repo's `Actions` tab to observe the progress of the deployment to production
-    - Note that the deployment job merely kicks off an ECS service update. To fully verify success, you'll need to check the
-      `Deployments` tab for the relevant service / cluster in the ECS console.
-  12. Send a message to `#researchnow_aka_sfr` in Slack to notify folks when the release finishes.
-
-And you're done!
-
-## TODO
-
-- Improve this README
-- ~~Add following data ingest processes:~~
-  - ~~NYPL Catalog~~
-  - ~~Project Gutenberg~~
-  - ~~DOAB~~
-  - ~~Project MUSE~~
-  - ~~MET Exhibition Catalogs~~
-- ~~Add centralized logging process~~
-- Add commenting/documentation strings
-- ~~Generate C4 diagrams for application~~
-- ~~Integrate ePub processor into standard processing flow~~
-- ~~Add cover fetching process~~
-- Create test suite, including:
-  - ~~Unit tests for all components~~
-  - Functional tests for each process
-  - Integration tests for the full cluster
-- Update dependencies
+1. Create new version branch (e.g. `v0.12.0`)
+2. Create PR titled "Release v0.12.0"
+3. After approval, squash and merge to main
+4. Create Github release:
+   - Tag: v0.12.0
+   - Target: main
+   - Generate notes: `cat <(git log <first-commit>..HEAD --pretty=format:"%h - %s (%an, %ad)" --date=short)`
+5. Notify team in #researchnow_aka_sfr Slack channel
+6. Monitor deployment in Github Actions and ECS console
```

Comment:
We can just say we are full CD - so maybe some details on what that process looks like.